server.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+require 'multi_json'
+require 'net/http'
+require 'uri'
+
+configure do
+  set :port, 5000
+end
+
+get '/' do
+  send_file 'index.html'
+end
+
+post '/user/verify' do
+  data = MultiJson.load(request.body.read)
+
+  url = URI.parse(data['apiUrl'])
+  req = Net::HTTP::Get.new(url.path)
+  req['Authorization'] = data['authHeader']
+
+  http = Net::HTTP.new(url.host, url.port)
+  http.use_ssl = true if url.scheme == 'https'
+  res  = http.request(req)
+
+  halt res.code.to_i, 'Error! Request failed.' unless res.code == '200'
+
+  MultiJson.dump(res.body)
```

Comment:
Sweet. All updated.

I added to this a little bit as well to make sure its always setting the right content-type. I made the ruby version do the exact same.

https://github.com/brandonblack/react-digits/commit/eb08a38eadddff5d3a64c4d245e1eb004343516d